### PR TITLE
Added health-check implementation and description

### PR DESCRIPTION
**Pull request by orozluk**

### CODE REVIEW

**Review by apenalver2 — CHANGES_REQUESTED** [review]
Can we include in the .gitignore file the test project obj?

File: src/main/Healthchecks/ExampleHealthCheck.cs [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    {
+        private readonly IConfigurationRoot configuration;
+
+        public ExampleHealthCheck(IConfigurationRoot configuration)
+        {
+            // Use DI to inject services here
+            this.configuration = configuration;            
+        }
+
+        public Task<HealthCheckResult> CheckHealthAsync(
```
Comment (PacmanForever) [inline src/main/Healthchecks/ExampleHealthCheck.cs:19]: If the class name is "ExampleHealthCheck", the method can rename to "CheckAsync", "GoAsync" or "ExecuteAsync".

File: src/main/Healthchecks/ExampleHealthCheck.cs [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+            HealthCheckContext context,
+            CancellationToken cancellationToken)
+        {
+            // Do your logic here 
+            bool healthCheckResultHealthy = this.configuration != null;
+
+            if (healthCheckResultHealthy)
```
Comment (PacmanForever) [inline src/main/Healthchecks/ExampleHealthCheck.cs:26]: Merge previous and this line.

File: src/main/Healthchecks/ExampleHealthCheck.cs (hunk 1 above)
Comment (orozluk) [reply src/main/Healthchecks/ExampleHealthCheck.cs:19]: It is just an example class. In any case. CheckAsync would be the best option.

File: src/main/Healthchecks/ExampleHealthCheck.cs (hunk 2 above)
Comment (orozluk) [reply src/main/Healthchecks/ExampleHealthCheck.cs:26]: Again, is an empty example health check. If in the future it has more content that would be improved

**Comment by orozluk** [comment]
> Can we include in the .gitignore file the test project obj?

@apenalver2  included in latest commit

**Pull request merged**